sites/fast-tooling-examples/app/example-native-elements-1.ts

```diff
@@ -1,4 +1,6 @@
 export default {
     id: "native-element-1",
     text: "An example of a native input interacting with the Message System.",
+    codeSandboxSrc:
```

Comment:
Should this name be abstracted and not specific to a service? `codeEmebed` or somthing?